### PR TITLE
Fixed laptopmag.com

### PR DESCRIPTION
**Pull request by sprinkledcheese**
- Logo inversion.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -1948,6 +1948,13 @@ INVERT
 
 ================================
 
+laptopmag.com
+
+INVERT
+div#publisherDetails.logo
+
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1955]: Add `.qc-cmp-publisher-logo` The cookie wall shows a logo as well.

Comment (sprinkledcheese) [reply src/config/dynamic-theme-fixes.config:1955]: Didn't have that banner since i'm using ubo. :)

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:1956]: uBlock Origin?

Comment (sprinkledcheese) [reply src/config/dynamic-theme-fixes.config:1956]: Yes.

**Pull request merged**